src/resources/js/Pages/Employee/Logs/Index.vue

```diff
@@ -139,39 +139,23 @@ export default {
       type: String,
       default: '',
     },
-    url: {
-      type: String,
-      default: null,
-    },
     paginator: {
       type: Object,
       default: null,
     },
   },
-  create() {
-    window.addEventListener("load", this.onWindowLoad);
-  },
   data() {
     return {
-      filterByLogType: '',
-      logTypes,
+      filterByLogType: this.search,
     };
   },
   computed: {
-    filteredLogs() {
-      if (!this.filterByLogType) return this.logs;
-      window.location.href = this.url + '?filterByLogType=' + this.filterByLogType;
-      return false;
+    filterLogs() {
+      Inertia.reload({ data: { filterByLogType: this.filterByLogType } });
```

Comment:
Inertia tries to make the website feel like an `SPA` by never fully reloading the webpage. To make this feel more seamless, we can use `Inertia`'s render functions to reload the page with a different query parameter. This makes it feel smoother and you get just the parts of the page that changed reloading